source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -18,12 +20,16 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace ZooKeeperProxy {
 
-ZooKeeperFilterConfig::ZooKeeperFilterConfig(const std::string& stat_prefix,
-                                             const uint32_t max_packet_bytes, Stats::Scope& scope)
+ZooKeeperFilterConfig::ZooKeeperFilterConfig(
+    const std::string& stat_prefix, const uint32_t max_packet_bytes,
+    const Protobuf::RepeatedPtrField<
+        envoy::extensions::filters::network::zookeeper_proxy::v3::LatencyThreshold>&
+        latency_thresholds,
+    Stats::Scope& scope)
     : scope_(scope), max_packet_bytes_(max_packet_bytes), stats_(generateStats(stat_prefix, scope)),
+      latency_threshold_map_(parseLatencyThresholds(latency_thresholds)),
       stat_name_set_(scope.symbolTable().makeSet("Zookeeper")),
       stat_prefix_(stat_name_set_->add(stat_prefix)), auth_(stat_name_set_->add("auth")),
-      connect_latency_(stat_name_set_->add("connect_response_latency")),
```

Comment:
To make all opcode has the same latency metric name format `[opcode]_resp_latency`